fabric-loot-api-v2/src/main/java/net/fabricmc/fabric/mixin/loot/LootManagerMixin.java

```diff
@@ -69,7 +69,7 @@ private void applyLootTableEvents(ResourceManager resourceManager, LootManager l
 				return;
 			}
 
-			LootTable table = (LootTable) entry;
+			if (!(entry instanceof LootTable table)) return;
```

Comment:
This isnt quite right, it should put the id + entry into `newTables` otherwise it will get lost.